src/midiEditorCommands.cpp

```diff
@@ -729,6 +729,23 @@ void postMidiSelectNotes(int command) {
 	outputMessage(s);
 }
 
+void postMidiSelectEvents(int command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int evtIndex=-1;
+	int count=0;
+	for(;;){
+		evtIndex = MIDI_EnumSelEvts(take, evtIndex);
+		if (evtIndex == -1)
+			break;
```

Comment:
nit: There's still a lot of older code that doesn't do this, but I'm trying to ensure newer code braces statement blocks even if there's only a single statement.